source/index.rst

```diff
@@ -7,11 +7,17 @@
 Welcome to Qucs Help's documentation!
 =====================================
 
+.. row of flags, alphabetical 
```

Comment:
This is now removed, flags was a bad idea. One languge can be used in many places